src/Chain.php

```diff
@@ -337,6 +337,26 @@ public function validate(MessageStack $messageStack, Container $input, Container
     }
 
     /**
+     * Get all rules applied to this chain
+     */
+    public function getRules()
+    {
+        $rules = [];
+
+        /** @var Rule $rule */
+        foreach ($this->rules as $rule) {
+            $reflectionRule = new \ReflectionClass($rule);
+            $rules[] = [
+                'name' => $reflectionRule->getShortName(),
```

Comment:
Reflection might not be the best solution here, any suggestions on improving this? Adding a constant for every class containing the class name seems overkill to me.